src/bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+// PHP_CodeSniffer does not define an autoloader in its composer.json,
+// so we must invoke it ourselves.
+require __DIR__ . "/../vendor/squizlabs/php_codesniffer/autoload.php";
```

Comment:
There was at the time a reason why I had to depend on the unstable version, it had an important bug fix - maybe it's fixed now in the most recent RC, might be worth to try out